.eslintrc

```diff
@@ -0,0 +1,130 @@
+{
```

Comment:
People at microapps use this config:
```json
{
  "env": {
    "mocha": true
  },
  "rules": {
    "comma-dangle": [2, "never"],
    "object-curly-spacing": 0,
    "no-shadow": 0
  },
  "extends":"airbnb/base"
}
```
but we adjust this at a later stage it's not very important right now.